src/index.js

```diff
@@ -20,10 +18,83 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+function render(state = {
+  leftNumber: 0,
+  rightNumber: 0,
+  operator: null,
+  result: 0,
+}) {
```

Comment:
```suggestion
function render({
  leftNumber: 0,
  rightNumber: 0,
  operator: null,
  result: 0,
}) {
```

처럼 매개변수 자체를 destructuring 해서 넣어줄 수 있어요.

Reply:
앗 그렇네요. 왜 이렇게 생각을 못했을까요 ㅠ